cachito/workers/tasks/npm.py

```diff
@@ -32,6 +32,51 @@
 log = logging.getLogger(__name__)
 
 
+def _verify_npm_files(bundle_dir, subpaths):
+    """
+    Verify that the correct are present for the npm package manager to proceed.
```

Comment:
Did you mean `correct npm files are present`?

Reply:
Maybe it should be `expected npm files` ?